### ISSUE TITLE
Spike: Determine Luminance and Acceptable Colors for Links/Whole Site

### ISSUE THREAD

**Issue by andyvanavery** [NCIOCPL/ncids#978]
### Expected Deliverables:
* Investigating changes to the luminance values to meet the AA and AAA a11y requirements
* If changes to luminance do not meet the requirements, investigate changing the colors chosen on the site to meet the a11y requirements

### Results of Spike
*Enter the information, or links to any (non-sensative) files with the findings here*

**Comment by karlikpj** [comment]
**Initial test** - Is to update our color theme tokens palettes to match the grade used by USWDS in those same tokens (ie primary_darker = blue-v60 uswds = navy-v60 ncids)

The result of correcting our palette to match uswds shows that in some cases we picked incorrectly from accent palettes that have a 20 pt less value than the primary and secondary palettes. 

**Second step** - validating that the colors we chose actually fit between the USWDS luminance values listed here https://designsystem.digital.gov/design-tokens/color/overview/

Our custom color tokens are correct and fit within the luminance values. So we know our values are correct and can now reverse engineer where colors are off.

So far besides buttons in general - most problems are around custom items and or section-dark specifically.
